types/chrome/index.d.ts

```diff
@@ -258,6 +263,16 @@ declare namespace chrome.action {
      */
     export function getUserSettings(): Promise<UserSettings>;
 
+    /**
+     * Since Chrome 99+.
+     * Opens the extension's popup.
+     * @param options Specifies options for opening the popup.
+     * @param callback The callback parameter should be a function that looks like this:
+     * () => {...}
+     * @return The `openPopup` method provides its result via callback or returned as a `Promise` (MV3 only). It has no parameters.
+     */
+    export function openPopup(options?: OpenPopupOptions, callback?: () => void): Promise<void>;
```

Comment:
I think we need an overload for `openPopup` here, a promise is returned only when the entire callback is omitted. We can probably follow the same pattern as `getUserSettings` above on line 257.

Reply:
You're right, I missed that